prow/spyglass/podlogartifact.go

```diff
@@ -144,7 +143,7 @@ func (a *PodLogArtifact) ReadAtMost(n int64) ([]byte, error) {
 
 // ReadTail reads the last n bytes of the pod log
 func (a *PodLogArtifact) ReadTail(n int64) ([]byte, error) {
-	logs, err := a.jobAgent.GetJobLogTail(a.name, a.buildID, n)
+	logs, err := a.jobAgent.GetJobLog(a.name, a.buildID)
```

Comment:
This is the more correct log tail implementation -- would honestly be better to refactor to be last `n` lines to match k8s API but /shrug